bananas_server/openttd/tcp_content.py

```diff
@@ -132,7 +132,9 @@ async def send_packet(self, data):
         if self.transport.is_closing():
             raise SocketClosed
 
-        self.transport.write(data)
+        res = self.transport.write(data)
+        if iscoroutine(res):
+            await res
```

Comment:
presume this is a false positive codeql notice

Reply:
Yes, it is. Wtf? Why would it think that ... what a silly CodeQL ...